### PR TITLE
#164798200 User should receive descriptive validation errors messages on registration

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

- Enables users receive descriptive error validation messages on signing up

#### Description of Task to be completed?
-  When a user is registering for an account and somehow they don't meet the requirements required for somebody to create an account, then they should be able to get a message as to why they are not able to fulfill their request

#### How should this be manually tested?

- Fetch this branch `git fetch origin ft-descriptive-validation-errors-164798200`
- Checkout to the branch `git checkout ft-descriptive-validation-errors-164798200`
- Run `python manage.py runserver`
- Run register user endpoint

#### What are the relevant pivotal tracker stories?
[#164798200](https://www.pivotaltracker.com/story/show/164798200)

#### Postman Collection
[Run in Postman](https://app.getpostman.com/run-collection/cb67ec77f04e89b40156)

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,2 +1 @@
-web: gunicorn authors.wsgi
-release: python manage.py migrate --noinput
\ No newline at end of file
+web: gunicorn authors.wsgi
```
Comment (ja-odur) [inline Procfile]: @felixkiryowa Please undo the changes you made to this filet o this  file.

File: Pipfile [hunk 2]
```diff
@@ -13,7 +13,8 @@ django-cors-headers = "*"
 djangorestframework = "*"
 django-extensions = "*"
 dj-database-url = "*"
-psycopg2-binary = "*"
+psycopg2 = "<=2.7.7"
```
Comment (ja-odur) [inline Pipfile]: @felixkiryowa undo changes made to thisfile.

**Review by ja-odur — DISMISSED** [review]

**Review by ezrogha — DISMISSED** [review]
Well done @felixkiryowa

**Review by ezrogha — DISMISSED** [review]

**Review by dorothyas — DISMISSED** [review]

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by ezrogha — APPROVED** [review]

**Review by dorothyas — APPROVED** [review]
